q2lint.py

```diff
@@ -27,6 +28,19 @@ def main():
             if header != HEADER:
                 errors.append('Invalid header: %s' % filepath)
 
+    npm_packages = filter(lambda x: 'node_modules' not in x,
```

Comment:
Let's keep it, it doesn't seem to hurt having it just in case there's local `node_modules` dirs during development.